frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -70,10 +75,10 @@ export const DataTablePlugin = createPlugin<S>("marimo-table")
       rowHeaders: z.array(z.tuple([z.string(), z.array(z.any())])),
       fieldTypes: z
         .record(
-          z.enum(["boolean", "integer", "number", "date", "string", "unknown"]),
+          z.enum(["boolean", "integer", "number", "date", "string", "unknown"])
         )
         .nullish(),
-    }),
+    })
```

Comment:
formatting seems off, you might want to run `cd frontend` ; `pnpm format` . also `pnpm lint` will show you the lint errors